package.json

```diff
@@ -27,65 +27,67 @@
   },
   "homepage": "https://github.com/angular-slider/ng5-slider#readme",
   "scripts": {
+    "ng": "ng",
     "prepare": "npm run prepare:lib && npm run prepare:demo-app",
-    "prepare:lib": "node scripts/generate-lib-files.js",
+    "prepare:lib": "ng build ng5-slider --prod",
     "prepare:demo-app": "node scripts/generate-demo-app-snippets.js && node scripts/generate-demo-app-docs.js",
     "start": "npm run prepare && ng serve",
-    "build": "npm run build:lib && npm run build:demo-app",
+    "build": "npm run build:demo-app",
     "build:lib": "npm run prepare && rimraf dist/ng5-slider dist/ng5-slider.tgz && ng-packagr -p src/ng5-slider/package.json && scss-bundle -e src/ng5-slider/lib/main.scss -d dist/ng5-slider/scss/ng5-slider.scss",
     "build:demo-app": "npm run prepare && rimraf dist/demo-app && ng build --prod --base-href /ng5-slider/",
     "publish-gh": "scripts/publish-gh-pages.sh",
-    "test": "npm run prepare && ng test",
-    "test:once": "npm run prepare && ng test --watch=false",
+    "test": "npm run prepare && ng test ng5-slider",
+    "test:once": "npm run prepare && ng test ng5-slider --watch=false",
     "lint": "npm run prepare && ng lint",
-    "e2e": "npm run prepare && ng e2e --environment=e2e",
-    "ci": "npm run test:once && npm run e2e && npm run lint && npm run build:lib && npm run build:demo-app"
+    "e2e": "npm run prepare && ng e2e",
+    "ci": "npm run test:once && npm run e2e && npm run lint && npm run prepare:lib && npm run build:demo-app"
   },
   "dependencies": {
-    "@angular/animations": ">=5.0.2",
-    "@angular/common": ">=5.0.2",
-    "@angular/compiler": ">=5.0.2",
-    "@angular/core": ">=5.0.2",
-    "@angular/forms": ">=5.0.2",
-    "@angular/http": ">=5.0.2",
-    "@angular/platform-browser": ">=5.0.2",
-    "@angular/platform-browser-dynamic": ">=5.0.2",
-    "@angular/router": ">=5.0.2",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0",
-    "core-js": "^2.4.1",
-    "detect-passive-events": "^1.0.4",
-    "rxjs": ">=5.5.6",
-    "rxjs-compat": "^6.5.2",
-    "zone.js": "^0.8.19"
+    "@angular/animations": "~9.0.0",
+    "@angular/common": "~9.0.0",
```

Comment:
I'm not a maintainer, but I doubt this has a chance to be merged - as it breaks backward compatibility with Angular 5+ version

Is there a way to fix the plugin while still keeping this compatibility?
What is the oldest Angular version that can be supported if we make it compatible with Angular 9?

Reply:
I expect that once this PR is merged, there will be a release of new major version (2.0.0) for this package with Angular 9+ support, and the current version (1.2.x) would be for the legacy Angular versions (v5 to v8). Also it would be great to replace **ng5** prefix with something more meaningful.